python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -26,3 +26,40 @@ def fit(m, x, y=None):
 
 def fit_transform(m, x):
     m.fit_transform(x)
+
+
+def predict(m, x, y=None):
+    m.predict(x) if y is None else m.predict(x, y)
+
+
+def _fil_classification_setup(m, data, arg={}):
+    from cuml.utils.import_utils import has_xgboost
+    if has_xgboost():
+        import xgboost as xgb
+    else:
+        raise ImportError("No XGBoost package found which is required for benchmarking FIL")
+    import os 
+
+    tmp_path = "./fil_models"
```

Comment:
Do you want to overwrite this each time or keep models around to be cached for future runs? If you want to overwrite each time (which I think is the desired behavior here), maybe better to create a temp directory with `mkdtemp` so we don't have collisions from multiple runs or other confusion?

Reply:
I was trying to cache the model so that we don't need to re-train it every time, but I found the setup function is called for every set of parameters which means a new model is trained for every run. As there is no support for caching for now, I'll use `mkdtemp` as you suggest to overwrite the model each time